### PR TITLE
Update VDC license msg

### PR DESCRIPTION
**Pull request by MElborolossy**
### Description

- Update the License message that appears to the user at the first login to the VDC.
- Update wiki link with https://wiki.cloud.threefold.io

### Related Issues

#2726 

### Screenshot
image

### Checklist

- [x] [Pre-commit hook is installed](https://github.com/threefoldtech/js-ng#pre-commit) to do formatting checks before committing code...etc
- [x] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings

### CODE REVIEW

File: jumpscale/packages/vdc_dashboard/frontend/components/License.vue [hunk 1]
```diff
@@ -37,12 +37,10 @@
       <v-card>
         <v-card-title class="headline"> Welcome to ThreeFold Now </v-card-title>
         <v-card-text>
-          This marketplace is a showcase of open source peer-to-peer apps built on top of the TF Grid. We are in demo mode
-          and running on testnet. Note your deployment will be cancelled automatically after three hours.
-          Forgive any instability you might encounter while our developers work out the kinks.
+          This marketplace is a showcase of open source peer-to-peer apps built on top of the TF Grid.
```
Comment (waleedhammam) [inline jumpscale/packages/vdc_dashboard/frontend/components/License.vue:40]: I think it's not not a showcase anymore ?

Comment (waleedhammam) [reply jumpscale/packages/vdc_dashboard/frontend/components/License.vue:40]: We can remove the whole message i think, this was intended to be a warning for solutions deletion

Comment (MElborolossy) [reply jumpscale/packages/vdc_dashboard/frontend/components/License.vue:40]: @xmonader what is your opinion?

Comment (xmonader) [reply jumpscale/packages/vdc_dashboard/frontend/components/License.vue:40]: @rkhamis can you please provide a better phrasing?

Comment (rkhamis) [reply jumpscale/packages/vdc_dashboard/frontend/components/License.vue:40]: the one @MElborolossy has is quite alright!

Comment (rkhamis) [reply jumpscale/packages/vdc_dashboard/frontend/components/License.vue:40]: the marketplace is a platform for opensource peer-to-peer apps built on top of TF grid


what do you think guys?

**Review by xmonader — APPROVED** [review]

**Pull request merged**